app/src/main/java/com/nutomic/syncthingandroid/service/RestApi.java

```diff
@@ -187,24 +186,14 @@ private void tryIsAvailable() {
         }
     }
 
-    /**
-     * Either shows a restart dialog, or only updates the config, depending on
-     * {@link #mRestartPostponed}.
-     */
-    public void showRestartDialog(Activity activity) {
-        if (mRestartPostponed) {
-            sendConfig();
-        } else {
-            activity.startActivity(new Intent(mContext, RestartActivity.class));
```

Comment:
Can the RestartActivity class be deleted? Together with the UI if there is any?

Reply:
Other than this, this looks good to me.

Reply:
Done, build okay https://github.com/syncthing/syncthing-android/pull/1099/commits/50622863287755c18f76978f60c88cbaae3dca77
Please note: A tx push is needed after that to get the unused strings correctly disposed in alle languages as we've discussed some days ago.